### ISSUE TITLE
Input Lara

### ISSUE THREAD

**Issue by jens-daniel-mueller** [jens-daniel-mueller/BloomSail#2]
Hi Lara,
please extract data from 2d- and 3d-GETM “E” within date range June 01 - Aug 31 2018 and within lat 57.3 - 57.5. We need the regional mean values for each time step and depth interval (in other words, we do not want to resolve latitudinal patterns within the chosen area).

Based on the extracted data, please produce 3 types of plots:
- [x] Hovmoeller plots: x: time; y: depth (0-80m); col: S, T, and rho; +add mld_rho as white line
- [x] Daily mean profile plots: x: S,T, and rho; y: depth, col: time
- [x] Time series wind speed; calculated from u and v component

I'm not sure if seawater density (rho) is included in the GETM files. If not, leave those plots open and I'll add code to calculate rho from S and T.

Please write all required code in analysis/Finnmaid+GETM.Rmd

Cheers!

**Comment by LSBurchardt** [comment]
Each time step means every 3-hour interval of a day, so 8 values per day?
Same for depth intervals, every unique depth value between 0-80 m is to be written?

**Comment by jens-daniel-mueller** [comment]
Adjustments are done!
I made more changes than expected.

In particular, I combined the reading of the different parameters from the 2d and 3d files into one for-loop. So, you only have to touch both files once, can easily add more parameters if needed and have saved 150 lines of code ;-) Wherever possible, we should also handle this in the further work in the Finnmaid-GETM project.

Daily mean profile plots are added and all if knitted&published.
